nncf/onnx/quantization/default_quantization.py

```diff
@@ -49,6 +50,7 @@
         ONNXBatchNormMetatype,
         ONNXHardSigmoidMetatype,
         ONNXResizeMetatype,
+        ONNXPowMetatype
```

Comment:
It it aligned with the `OpenVINO` backend? Especially, `SQRT` quantization.
cc @l-bat

Reply:
I can align it in https://github.com/openvinotoolkit/nncf/pull/1709

Reply:
Aligned with OV and the PR https://github.com/openvinotoolkit/nncf/pull/1709